app/controllers/mixins/containers_external_logging_support_mixin.rb

```diff
@@ -0,0 +1,17 @@
+module ContainersExternalLoggingSupportMixin
+  def launch_external_logging_support
+    record = self.class.model.find(params[:id])
+    ems = record.ext_management_system
+    route_name = ems.external_logging_route_name
+    logging_route = ContainerRoute.find_by(:name => route_name, :ems_id => ems.id) if route_name
+    if logging_route
+      url = "https://#{logging_route.host_name}#{record.external_logging_path}"
```

Comment:
@simon3z My routes don't have a http/https specified in them. I guess this is optional to them. I can check for it and use the route's protocol if it is mentioned.